examples/jsm/nodes/shadernode/ShaderNodeElements.js

```diff
@@ -65,8 +65,7 @@ export const cubeTexture = nodeProxy( CubeTextureNode );
 
 export const instance = nodeProxy( InstanceNode );
 
-export const reflectVector = nodeImmutable( ReflectNode, ReflectNode.VECTOR );
-export const reflectCube = nodeImmutable( ReflectNode, ReflectNode.CUBE );
+export const reflectVector = nodeImmutable( ReflectNode );
```

Comment:
Maybe it is better to rename it to just `reflect`?

Reply:
maybe rename the class for `ReflectVectorNode` because `reflect` is a math function.